core/schema/sdk.go

```diff
@@ -616,6 +702,15 @@ func (sdk *goSDK) Runtime(
 		return nil, err
 	}
 	if err := sdk.dag.Select(ctx, ctr, &ctr,
+		dagql.Selector{
+			Field: "withoutUnixSocket",
+			Args: []dagql.NamedInput{
+				{
+					Name:  "path",
+					Value: dagql.NewString("/tmp/dagger-ssh-sock"),
+				},
+			},
+		},
```

Comment:
:eyes: not sure I get why we're doing this.

Reply:
I need to double check this one. I think I cleanedup all the weird changes I had to do to make ssh auth sockets work, and this was required (and also because Erik suggested we should remove the socket from runtime). I will double check and confirm before tagging Erik for his feedback.

Reply:
I revisited my notes, and here is the comment from Erik where he suggested this to fix socket not found issue. His suggestion was both from the fix of the issue as well as from security perspective like commented here: https://github.com/rajatjindal/dagger/blob/6ce731c74c119cd83923b7c221f414f25cd7ef06/core/schema/sdk.go#L538-L538

https://discord.com/channels/707636530424053791/1342188325712429187/1342252739551297798